tests/app/views/contexts/summary/test_question.py

```diff
@@ -1,1114 +1,1014 @@
-        self.assertEqual(question.answers[0]["value"][1]["label"], "Dark Side label")
-
-    def test_checkbox_button_detail_answer_empty(self):
-        # Given
-        self.answer_store.add_or_update(
-            Answer(answer_id="answer_1", value=["other", ""])
-        )
-
-        options = [
-            {"label": "Light Side", "value": "Light Side"},
-            {
-                "label": "Other option label",
-                "value": "other",
-                "other": {"label": "Please specify other"},
-            },
-        ]
-        answer_schema = {
-            "id": "answer_1",
-            "label": "Which side?",
-            "type": "Checkbox",
-            "options": options,
-        }
-        question_schema = {
-            "id": "question_id",
-            "title": "question_title",
-            "type": "GENERAL",
-            "answers": [answer_schema],
+            ]
         }
-
-        # When
-        question = Question(
-            question_schema,
-            answer_store=self.answer_store,
-            schema=self.schema,
-            rule_evaluator=self.get_rule_evaluator(),
-            value_source_resolver=self.get_value_source_resolver(),
-            location=None,
-            block_id="house-type",
-            return_to=None,
-        )
-
-        # Then
-        self.assertEqual(len(question.answers[0]["value"]), 1)
-        self.assertEqual(question.answers[0]["value"][0]["label"], "Other option label")
-        self.assertEqual(question.answers[0]["value"][0]["detail_answer_value"], None)
-
-    def test_checkbox_answer_with_detail_answer_returns_the_value(self):
-        # Given
-        self.answer_store.add_or_update(
-            Answer(answer_id="answer_1", value=["Light Side", "Other"])
-        )
-        self.answer_store.add_or_update(Answer(answer_id="child_answer", value="Test"))
-
-        options = [
-            {"label": "Light Side", "value": "Light Side"},
+    )
+
+
+def address_question(answer_store, list_store, schema):
+    question_schema = schema.get_questions("what-is-your-address-question")[0]
+    return Question(
+        question_schema,
+        answer_store=answer_store,
+        schema=schema,
+        rule_evaluator=get_rule_evaluator(answer_store, list_store, schema),
+        location=None,
+        block_id="address-block",
+        return_to=None,
+        value_source_resolver=get_value_source_resolver(
+            answer_store, list_store, schema
+        ),
+    )
+
+
+@pytest.mark.usefixtures("app")
+@pytest.mark.parametrize(
+    "question_title,answers,expected_title,expected_len",
+    (
+        ("question_title", [MagicMock()], "question_title", 1),
+        ("question_title", [], "question_title", 0),
+        (
+            "",
+            [{"type": "Number", "id": "age-answer", "mandatory": True, "label": "Age"}],
+            "Age",
+            1,
+        ),
+    ),
```

Comment:
Existing issue, but it would help if question title was realistic, being snake case is confusing as I think it is a property etc. Maybe just call it like `Some title` etc.

Reply:
Couple more remaining, I didn't highlight all.